articles/quickstart/native/ios-swift/interactive.md

```diff
@@ -0,0 +1,223 @@
+---
+title: Add Login to your iOS or macOS app
+description: This guide demonstrates how to add authentication and gain access to user profile information in any iOS / macOS app using the Auth0.swift SDK.
+interactive: true
+files:
+  - files/Auth0
+  - files/ContentView
+  - files/ProfileView
+  - files/User
+seo_alias: swift
+budicon: 448
+topics:
+  - quickstarts
+  - native
+  - ios
+  - macos
+  - swift
+github:
+  path: Sample-01
+contentType: tutorial
+useCase: quickstart
+---
+
+# Add Login to your iOS or macOS app
+
+This guide demonstrates how to add authentication and gain access to user profile information in any iOS / macOS app using the [Auth0.swift](https://github.com/auth0/Auth0.swift) SDK.
+
+To use this quickstart, you’ll need to:
+* Sign up for a free Auth0 account or log in to Auth0.
+* Have an existing iOS / macOS app that you want to integrate with. Alternatively, you can view or download a sample app after logging in.
+
+## Configure Auth0 {{{ data-action=configure }}}
+
+To use Auth0 services, you’ll need to have an application set up in the Auth0 Dashboard. The Auth0 application is where you will configure how you want authentication to work for the app you are developing.
+
+### Configure an Auth0 Application
```

Comment:
Style guide (word list): titles should be sentence case and "application" should not be capitalized

```suggestion
### Configure an Auth0 application
```